kolibri/core/device/test/test_api.py

```diff
@@ -862,3 +862,39 @@ def test_usersyncstatus_list__unknown_non_negative_device_status(self):
         self.assertEqual(len(response.data), 1)
         self.assertEqual(response.data[0]["user"], self.user1.id)
         self.assertEqual(response.data[0]["status"], user_sync_statuses.RECENTLY_SYNCED)
+
+    def test_has_downloads(self):
+        self.client.login(
+            username=self.user1.username,
+            password=DUMMY_PASSWORD,
+            facility=self.facility,
+        )
+        self._create_transfer_session(
+            active=False,
+        )
+        self.syncsession1.last_download = None
+        self.syncsession1.save()
+
+        response = self.client.get(reverse("kolibri:core:usersyncstatus-list"))
+        self.assertEqual(len(response.data), 1)
+        self.assertEqual(response.data[0]["status"], user_sync_statuses.RECENTLY_SYNCED)
+
+    def test_last_download_removed(self):
+        self.client.login(
+            username=self.user1.username,
+            password=DUMMY_PASSWORD,
+            facility=self.facility,
+        )
+        self._create_transfer_session(
+            active=False,
+        )
+        self.syncsession1.last_download = None
+        self.syncsession1.status = (
+            user_sync_statuses.RECENTLY_SYNCED
+        )  # Set status to QUEUED
```

Comment:
This `self.syncsession1` is a `SyncSession` model and does not have a `status` field. I'm unsure what your intention is here?